packages/orm/src/test/helpers.ts

```diff
@@ -320,13 +335,13 @@ export type Schema = {
   Genre: typeof Genre;
   Tag: typeof Tag;
   Author: typeof Author;
-  Movie: typeof Movie; 
+  Movie: typeof Movie;
   Publisher: typeof Publisher;
-  
-  BookGenres: typeof Model;
-  BookTags: typeof Model;
+
+  BookGenres: typeof AnyModel;
+  BookTags: typeof AnyModel;
   TagSubTags: typeof Tag;
-}
+};
```

Comment:
I am not wrapping the Schema type into `ValidateSchema` here as I am using AnyModel types for intermediate models created for MM relation purposes.